Automattic/jetpack
# Jetpack Network Update: Add a way to update the network to a new version

This PR lets sites update to the latest version of jetpack when the network changes.

This adds a bit of variability and lets us trigger full sync.
We are pausing this PR for now.

Do do.
- [ ] Clean up the network admin UI. (Header and footer)
- [ ]  Add more security. (ajax nonce and permission check)
- [ ] Add a button for the network admin to get to the new network admin section after they update Jetpack to the latest version.
- [ ] Add sync tests.
- [ ] Bump up the number of sites that get process per page. 
- [ ] Show some sort of indicator how far along we are? 

Some screenshots of how things look like right now.
![screen shot 2017-09-22 at 8 11 19 am](https://user-images.githubusercontent.com/115071/30751517-5fc16532-9f6e-11e7-90c5-6ab503f3a5c1.png)
![screen shot 2017-09-22 at 8 11 25 am](https://user-images.githubusercontent.com/115071/30751515-5fb90fa4-9f6e-11e7-8a35-c1fa5037b885.png)
![screen shot 2017-09-22 at 8 11 41 am](https://user-images.githubusercontent.com/115071/30751516-5fbcdf3a-9f6e-11e7-80ea-6513258d39ca.png)


#### Changes proposed in this Pull Request:
* 

#### Testing instructions:
* On the mu install. Update to this branch. Update the Jetpack version. Make sure that no full sync actions happen as you visit the pages. 
Go to the Network Update Network section. Click on the Button. Watch the full sync happen across all your network sites.


#### Proposed changelog entry for your changes:
